pagefaultgames/pokerogue
# [Bug] skip-seen-dialogue storing strings instead of i18n keys

## What are the changes the user will see?

Their previously stored seen dialogues will still apply correctly.

## Why am I making these changes?

Skip Seen dialogues used to store the i18n keys but I've accidentally changed that to the actual string. Meaning if you change the language all dialogues are considered "not seen" 

## What are the changes from a developer perspective?

I've made sure that skipSeenDialogues stores & checks the i18n keys instead of the actual resolved string

### Screenshots/Videos


https://github.com/user-attachments/assets/920d8ca4-105a-4884-8312-71c43aba78c1



## How to test the changes?
- Make sure to skip seen dialogues (settings -> general)
- Set starting wave to wave 5.
- Wait for the first dialogues
- reload
- start same game again
- YOu shouldn't see the dialogue this time
- change language
- reload
- same again same result -> shouldn't see it.
- Check the localstorage -> seenDialogues key

## Checklist
    - I tried but it was an absolute pain the a** as dialogues are skipped by default in tests
